examples/plasma_stick/plasma_stick_rainbows.cpp

```diff
@@ -34,10 +34,15 @@ int main() {
   while(true) {
 
     offset += float(SPEED) / 2000.0f;
+    if (offset > 1.0) {
+      offset -= 1.0;
+    }
```

Comment:
```suggestion
    offset %= 1
```

Reply:
Following up on https://github.com/LaaZa comment...

Reply:
this isn't legal code.

Reply:
You are totally right, the code proposal would then be only for the corresponding Python code, if you won't mind fixing the issue also over there :)